### PR TITLE
New package: Craftware.Keyhac version 1.82

### PR DESCRIPTION
**Pull request by kenzillow**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122887)

### CODE REVIEW

**Comment by kenzillow** [comment]
@microsoft-github-policy-service agree

File: manifests/c/Craftware/Keyhac/1.82/Craftware.Keyhac.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Craftware.Keyhac
+PackageVersion: "1.82"
+PackageLocale: en-US
+Publisher: Craftware
+PackageName: Keyhac
```
Comment (stephengillie) [inline manifests/c/Craftware/Keyhac/1.82/Craftware.Keyhac.locale.en-US.yaml:8]:
```suggestion
PackageName: Keyhac
PackageUrl: https://crftwr.github.io/keyhac
```

**Comment by stephengillie** [comment]
image

File: manifests/c/Craftware/Keyhac/1.82/Craftware.Keyhac.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Craftware.Keyhac
+PackageVersion: "1.82"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: keyhac\keyhac.exe
```
Comment (stephengillie) [inline manifests/c/Craftware/Keyhac/1.82/Craftware.Keyhac.installer.yaml:9]:
```suggestion
- RelativeFilePath: keyhac\keyhac.exe
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**